render/egl.c

```diff
@@ -248,28 +265,28 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 			"eglUnbindWaylandDisplayWL");
 		load_egl_proc(&egl->procs.eglQueryWaylandBufferWL,
 			"eglQueryWaylandBufferWL");
+	} else {
+		if (platform == EGL_PLATFORM_DEVICE_EXT) {
+			wlr_log(WLR_INFO, "EGL_WL_bind_wayland_display not supported");
```

Comment:
Please indicate that this is about the device platform and that we'll fallback to the GBM platform.

Reply:
Debug or info level

Reply:
I'll add the message in `wlr_egl_create_from_drm_fd`